### ISSUE TITLE
Share button

### ISSUE THREAD

**Issue by djibaout** [totorobin/poker#4]
Let's have the share button working !

**Comment by totorobin** [comment]
```
index-602a36d7.js:1 TypeError: Cannot read properties of undefined (reading 'writeText')
    at c (index-602a36d7.js:17:27193)
    at fn (index-602a36d7.js:1:13375)
    at mt (index-602a36d7.js:1:13454)
    at HTMLDivElement.n (index-602a36d7.js:1:56892)
```

**Comment by totorobin** [comment]
this has been fix with #10
